src/main/java/org/privacyidea/PIResponse.java

```diff
@@ -39,6 +39,7 @@
     public String jsonRPCVersion = "";
     public boolean status = false;
     public boolean value = false;
+    public String authentication = "";
```

Comment:
this should be of type AuthenticationStatus, not string

Reply:
And should be initialized to the default case (like "NONE")

Reply:
fixed